config/quipucords-app.container

```diff
@@ -19,7 +19,7 @@ Network=quipucords.network
 # Since we could write self-generated keys to the shared certs
 # volume and write nginx logs to the common log volume,
 # let's make sure we map the nginx user to the host's user.
-UserNS=keep-id:uid=1001,gid=101
```

Comment:
Huh. I'm surprised this low-number `gid` didn't cause major problems already. What specifically prompted this change?

Reply:
I just saw it was incorrect. To confirm this was a typo I ran `id` in the nginx image